tests/WalletMgr.py

```diff
@@ -12,29 +12,29 @@
 Wallet=namedtuple("Wallet", "name password host port")
 # pylint: disable=too-many-instance-attributes
 class WalletMgr(object):
-    __walletLogOutFile="test_keosd_out.log"
-    __walletLogErrFile="test_keosd_err.log"
-    __walletDataDir="test_wallet_0"
+    __walletLogOutFile="test_keosd_00000_out.log"
+    __walletLogErrFile="test_keosd_00000_err.log"
+    __walletDataDir="test_wallet_00000"
     __MaxPort=9999
 
     # pylint: disable=too-many-arguments
     # walletd [True|False] True=Launch wallet(keosd) process; False=Manage launch process externally.
-    def __init__(self, walletd, nodeosPort=8888, nodeosHost="localhost", port=9899, host="localhost", clusterID=0):
+    def __init__(self, walletd, nodeosPort=-1, nodeosHost="localhost", port=-1, host="localhost", clusterID=0):
         self.walletd=walletd
         self.nodeosPort=nodeosPort
         self.nodeosHost=nodeosHost
         self.port=port
         self.host=host
         self.wallets={}
         self.__walletPid=None
-        if clusterID != 0:
-            WalletMgr.__walletLogOutFile = "test_keosd_cluster" + str(clusterID) + "_out.log"
-            WalletMgr.__walletLogErrFile="test_keosd_cluster" + str(clusterID) + "_err.log"
-            WalletMgr.__walletDataDir="test_wallet_" + str(clusterID) 
-            if self.port == 9899:
-                self.port += clusterID     
-            if self.nodeosPort == 8888:
-                self.nodeosPort += clusterID * 2000
+
+        WalletMgr.__walletLogOutFile = "test_keosd_cluster" + ("%05d" % clusterID) + "_out.log"
```

Comment:
I would again prefer to see all cluster specific files to be contained in the same directory rather than augmenting the files names with the cluster IDs.